lib/icingadb/icingadb-objects.cpp

```diff
@@ -1851,7 +1854,6 @@ void IcingaDB::SendStartedDowntime(const Downtime::Ptr& downtime)
 		"flexible_duration", Convert::ToString(TimestampToMilliseconds(std::max(0.0, downtime->GetDuration()))),
 		"scheduled_start_time", Convert::ToString(TimestampToMilliseconds(downtime->GetStartTime())),
 		"scheduled_end_time", Convert::ToString(TimestampToMilliseconds(downtime->GetEndTime())),
-		"has_been_cancelled", Convert::ToString((unsigned short)downtime->GetWasCancelled()),
```

Comment:
Looks like you have cherry-picked the wrong version of https://github.com/Icinga/icinga2/pull/9896.